src/dungeon_ai.c

```diff
@@ -327,25 +441,25 @@ u8 sub_8075BF4(struct DungeonEntity * pokemon, s32 sleepTurns)
   return uVar4;
 }
 
-void sub_8075C58(struct DungeonEntity * pokemon, struct DungeonEntity * target, s32 param_3, u8 param_4)
+void sub_8075C58(struct DungeonEntity * pokemon, struct DungeonEntity * target, s32 turns, u8 displayMessage)
```

Comment:
The functions below don't look like AI code anymore. Perhaps we can move them to a different file.

Reply:
Might just do the same as what I say in the comment below (splitting to a `code_` file)

Reply:
The functions after `DecideAction()` are mostly labeled, so there's probably enough info to use a more descriptive name like `move_effects_target.c`.

Reply:
Split into `move_effects_target.c` as requested